cmd/commands/account/command.go

```diff
@@ -111,6 +120,41 @@ type command struct {
 	tequilapi *tequilapi_client.Client
 }
 
+func (c *command) setIdentity(ctx *cli.Context) {
+	givenID := ctx.Args().First()
+	if givenID == "" {
+		clio.Warn("No identity provided")
+		return
+	}
+
+	identities, err := c.tequilapi.GetIdentities()
+	if err != nil {
+		clio.Error("Failed to get existing identities")
+		return
+	}
+
+	found := false
+	for _, id := range identities {
+		if id.Address == givenID {
+			found = true
+			break
+		}
+	}
+
+	if !found {
```

Comment:
You dont need to do all this validation, it's performed by `CurrentIdentity()`

Reply:
I let myself be liberal with the calls because we're calling localhost after all and this is nicer in terms of displaying the actual error message without leaking anything that would confuse a simple non technical user. :shrug: